### PR TITLE
KEYCLOAK-11941 7.3.6 Rebase RH-SSO z-stream image stream on top of EAP z-stream image

### PR DESCRIPTION
**Pull request by drichtarik**
KEYCLOAK-12504 Remove S390 architecture from delivered images
@pskopek @iankko

### CODE REVIEW

File: image.yaml [hunk 1]
```diff
@@ -59,7 +59,7 @@ modules:
           - name: jboss-eap-modules
             git:
                   url: https://github.com/jboss-container-images/jboss-eap-modules.git
-                  ref: EAP_725_CR2
+                  ref: EAP_730_CR1
```
Comment (hmlnarik) [inline image.yaml]: Certainly this needs to be some EAP_726* tag

File: container.yaml [hunk 2]
```diff
@@ -4,4 +4,3 @@ compose:
 platforms:
   only:
   - x86_64
-  - s390x
```
Comment (hmlnarik) [inline container.yaml:7]: Do not remove, only comment out this line.

**Review by hmlnarik — CHANGES_REQUESTED** [review]

**Review by hmlnarik — APPROVED** [review]
Thanks @drichtarik!

File: image.yaml [hunk 3]
```diff
@@ -59,7 +59,7 @@ modules:
           - name: jboss-eap-modules
             git:
                   url: https://github.com/jboss-container-images/jboss-eap-modules.git
-                  ref: EAP_725_CR2
```
Comment (iankko) [inline image.yaml:62]: @drichtarik Maybe the cct_module above would be bumpable to [`0.39.0`](https://github.com/jboss-openshift/cct_module/releases/tag/0.39.0) version too?

Comment (drichtarik) [reply image.yaml:62]: OK. I didn't notice the tag since I checked the releases tab only.

Comment (iankko) [reply image.yaml:62]: Thanks

**Review by iankko — APPROVED** [review]

**Pull request merged**